### PR TITLE
[ISSUE #5243] Add acl support to TimerProducer/TimerConsumer

### LINKED ISSUE #5243: Add acl support to TimerProducer/TimerConsumer
**FEATURE REQUEST**

1. Please describe the feature you are requesting.

add acl support to TimerProducer/Consumer and fix some bugs

### PR DESCRIPTION
**Pull request by zhiliatom**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

close #5243 

## Brief changelog

1. fix wrong option name of namesrv
2. add acl support to TimerProducer/Consumer benchmark
3. add new options

### CODE REVIEW

**Review by echooymxq — DISMISSED** [review]

File: example/src/main/java/org/apache/rocketmq/example/benchmark/timer/TimerConsumer.java [hunk 1]
```diff
@@ -151,6 +166,30 @@ private Options buildCommandlineOptions(Options options) {
         opt.setRequired(false);
         options.addOption(opt);
 
+        opt = new Option("w", "threadCount", true, "Thread count, Default: 20");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("m", "msgTraceEnable", true, "Message Trace Enable, Default: false");
```
Comment (ltamber) [inline example/src/main/java/org/apache/rocketmq/example/benchmark/timer/TimerConsumer.java:173]: how about use a no argument option ?

File: example/src/main/java/org/apache/rocketmq/example/benchmark/timer/TimerConsumer.java [hunk 2]
```diff
@@ -151,6 +166,30 @@ private Options buildCommandlineOptions(Options options) {
...
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("g", "group", true, "Consumer group name, Default: benchmark_consumer");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("a", "aclEnable", true, "Acl Enable, Default: false");
```
Comment (ltamber) [inline example/src/main/java/org/apache/rocketmq/example/benchmark/timer/TimerConsumer.java:181]: how about use a no argument option ?

Comment (zhiliatom) [reply example/src/main/java/org/apache/rocketmq/example/benchmark/timer/TimerConsumer.java:181]: Good suggestion, but to keep consistent wist other code, I don't do any change here.
 If It really need to change, I'd like to make a new patch to change all .sh .

File: example/src/main/java/org/apache/rocketmq/example/benchmark/timer/TimerConsumer.java [hunk 3]
```diff
@@ -151,6 +166,30 @@ private Options buildCommandlineOptions(Options options) {
...
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("ak", "accessKey", true, "Acl access key, Default: rocketmq2");
```
Comment (lizhanhui) [inline example/src/main/java/org/apache/rocketmq/example/benchmark/timer/TimerConsumer.java:185]: Having defaults for access key/secret might not be a good idea.

Comment (zhiliatom) [reply example/src/main/java/org/apache/rocketmq/example/benchmark/timer/TimerConsumer.java:185]: Thanks for your review, I’ll exchange their default value like existed code.

**Review by HScarb — DISMISSED** [review]

**Review by complone — APPROVED** [review]

**Review by tsunghanjacktsai — APPROVED** [review]

**Review by yuz10 — APPROVED** [review]

**Pull request closed without merging**